internal/command/plan_test.go

```diff
@@ -220,6 +220,44 @@ func TestPlan_noState(t *testing.T) {
 	}
 }
 
+func TestPlan_noTestVars(t *testing.T) {
+	td := t.TempDir()
+	testCopyDir(t, testFixturePath("plan-no-test-vars"), td)
+	defer testChdir(t, td)()
+
+	p := planFixtureProvider()
+	view, done := testView(t)
+	c := &PlanCommand{
+		Meta: Meta{
+			testingOverrides: metaOverridesForProvider(p),
+			View:             view,
+		},
+	}
+
+	outPath := filepath.Join(td, "test.plan")
+	args := []string{
+		"-out", outPath,
+	}
+	code := c.Run(args)
+	output := done(t)
```

Comment:
As an alternative approach, you can check the output of the plan command instead of the plan file. You would want to check if `test_instance.testRes.ami` equals to expected value (`ValueFROMmain/tfvars`) in the output. This alternative approach could simplify the test a bit.

Reply:
I've replaced this with a simple `contains` in the output string, as I don't want to dictate any formatting requirements in the output
Given the test case, there won't be anything else that would assume the value `valueFROMmain/tfvars`so I think this is okay - let me know if you think we need to be more specific

Reply:
I don't think it is required, but I would use a more specific line to search for. Also, I would double-check if the output contains `ValueFROMtests/tfvars` just to be sure it doesn't.

Reply:
Updated to assert for the line `ami = "ValueFROMmain/tfvars"`
Added a check to ensure `ValueFROMtests/tfvars` doesn't exist in output